user-manual/access-content/browse.rst

```diff
@@ -234,6 +234,50 @@ the end of the list:
    Results may not be exactly as expected. Alphabetic sort order in AtoM is
    'ASCII-betical'. For more, see :ref:`navigate`.
 
+If an administrator has enabled the **Hierarchy browse** options (see
+:ref:`settings`) the user will also see a button to engage that view.
+
+.. image:: images/browse-hierarchy-choice.*
+   :align: center
+   :width: 70%
+   :alt: Browse hierarchy choice
+
+Clicking this button will take users to a full-page, full-width
+treeview containing all top-level descriptions in AtoM.
+
+.. image:: images/browse-hierarchy.*
+   :align: center
+   :width: 70%
+   :alt: Browse hierarchy page
+
+As with the current description treeview, each description in the hierarchy
+browse page that has children will display an “expand/collapse” icon which can
+be clicked to display all the children of that description. Any child
+descriptions that in turn have children, will also be expandable.
+
+A number in the top right corner of the page will indicate how many more
+descriptions can be seen in the hierarchy. Click on that number to bring up the
+next page of results.
+
+.. image:: images/browse-hierarchy-more.*
+   :align: center
+   :width: 70%
+   :alt: Browse hierarchy page
+
+The Browse by hierarchy treeview will have the same functionality as the
+standard treeview, except:
+
+* All description titles from any level in the catalogue database will be shown
```

Comment:
Everything looks good, but this was the one part that I wasn't sure about. I read this sentence as implying that even child level descriptions will appear alphabetically in the hierarchy browser - is this true, or do child records retain their original order? Might be worth double-checking? If it's true, then no changes needed! If not, maybe just clarify this or state that the sort order is alphabetic for top-level descriptions?

Reply:
good catch! I have updated the text